index.bs

```diff
@@ -299,9 +298,9 @@ The <dfn method for=FileSystemFileHandle>getFile()</dfn> method steps are:
   1. Let |f| be a new {{File}}.
   1. Set |f|'s <a spec=FileAPI>snapshot state</a> to the current state of |entry|.
   1. Set |f|'s underlying byte sequence to a copy of |entry|'s [=binary data=].
-  1. Initialize the value of |f|'s {{File/name}} attribute to |entry|'s [=entry/name=].
-  1. Initialize the value of |f|'s {{File/lastModified}} attribute to |entry|'s [=file entry/modification timestamp=].
-  1. Initialize the value of |f|'s {{Blob/type}} attribute to an [=implementation-defined=] value, based on for example |entry|'s [=entry/name=] or its file extension.
+  1. Set |f|.{{File/name}} to |entry|'s [=entry/name=].
+  1. Set |f|.{{File/lastModified}} to |entry|'s [=file entry/modification timestamp=].
+  1. Set |f|.{{Blob/type}} to an [=implementation-defined=] value, based on for example |entry|'s [=entry/name=] or its file extension.
```

Comment:
These setters follow the "Set f.name to entry's name" pattern as opposed to the "Set f's name to entry's name" pattern to match https://w3c.github.io/FileAPI/#dfn-file

That being said, lines 299 and 300 make a lot more sense in the current pattern (there's no ByteSequence definition to link to, AFAIK).... so maybe we should make this consistent and use the latter pattern everywhere here?

Reply:
I think what you have here currently is okay. The larger problem here is that the File API doesn't do a good job of defining the model of `File` object so you cannot manipulate the internal fields directly, which is what you'd want to do here.